main.tf

```diff
@@ -1 +1,79 @@
+resource "azurerm_firewall_policy_rule_collection_group" "policy_collection" {
```

Comment:
check naming of resources : https://confluence.one4all.lan/pages/viewpage.action?spaceKey=AMS&title=Terraform+directive#Terraformdirective-Resources

Reply:
Changed to "this"